kratos/kratos.go

```diff
@@ -0,0 +1,156 @@
+//
+// Copyright 2021 SkyAPM org
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package kratos
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/SkyAPM/go2sky"
+	"github.com/go-kratos/kratos/v2/log"
+	"github.com/go-kratos/kratos/v2/metadata"
+	"github.com/go-kratos/kratos/v2/middleware"
+	"github.com/go-kratos/kratos/v2/transport"
+	agentv3 "skywalking.apache.org/repo/goapi/collect/language/agent/v3"
+)
+
+const (
+	componentIDGoKratosServer = 5010
+	componentIDGoKratosClient = 5011
```

Comment:
@kagaya85  The `componentID` needs to be define in the [component-libraries.yml](https://github.com/apache/skywalking/blob/master/oap-server/server-bootstrap/src/main/resources/component-libraries.yml#L478).

Reply:
And here https://github.com/apache/skywalking-rocketbot-ui/tree/master/src/views/components/topology/assets

Reply:
@kagaya85 Let's merge these two, and you could pick one component ID.